### PR TITLE
Restart pricenode service every 24 hours.

### PR DESCRIPTION
**Pull request by ghost**
Implements one of the suggestions made in issue #33.
Restart is done automatically by the service config.

### CODE REVIEW

**Comment by gabernard** [comment]
@alvasw Could you please review this PR? Thank you.

File: scripts/bisq-pricenode.service [hunk 1]
```diff
@@ -7,7 +7,8 @@ SyslogIdentifier=bisq-pricenode
 EnvironmentFile=/etc/default/bisq-pricenode.env
 ExecStart=/bisq/bisq-pricenode/bisq-pricenode 2
 ExecStop=/bin/kill -TERM ${MAINPID}
-Restart=on-failure
+RuntimeMaxSec=86400
```
Comment (alvasw) [inline scripts/bisq-pricenode.service]: I think it's more readable if we use `RestartSec=24hr`.
```
       RestartSec=
           Configures the time to sleep before restarting a service (as configured with Restart=). Takes a
           unit-less value in seconds, or a time span value such as "5min 20s". Defaults to 100ms.
```

Valid time span values are:
```
SYSTEMD.TIME(7)                                    systemd.time                                   SYSTEMD.TIME(7)

NAME
       systemd.time - Time and date specifications

DESCRIPTION
       In systemd, timestamps, time spans, and calendar events are displayed and may be specified in closely
       related syntaxes.

DISPLAYING TIME SPANS
       Time spans refer to time durations. On display, systemd will present time spans as a space-separated
       series of time values each suffixed by a time unit. Example:

           2h 30min

       All specified time values are meant to be added up. The above hence refers to 150 minutes. Display is
       locale-independent, only English names for the time units are used.

PARSING TIME SPANS
       When parsing, systemd will accept the same time span syntax. Separating spaces may be omitted. The
       following time units are understood:

       •   usec, us, µs

       •   msec, ms

       •   seconds, second, sec, s

       •   minutes, minute, min, m

       •   hours, hour, hr, h

       •   days, day, d

       •   weeks, week, w

       •   months, month, M (defined as 30.44 days)

       •   years, year, y (defined as 365.25 days)
```

**Review by alvasw — CHANGES_REQUESTED** [review]
Nit

**Review by alvasw — APPROVED** [review]
utACK

**Review by gabernard — APPROVED** [review]
utACK

**Pull request merged**